packages/sveltekit/src/worker/handle.ts

```diff
@@ -0,0 +1,56 @@
+import { CloudflareOptions, continueTrace, wrapRequestHandler } from '@sentry/cloudflare';
+import { getDefaultIntegrations as getDefaultCloudflareIntegrations } from '@sentry/cloudflare';
+import type { Handle } from '@sveltejs/kit';
+
+import { sentryHandleGeneric, SentryHandleOptions } from '../server-common/handle';
+import { rewriteFramesIntegration } from '../server-common/rewriteFramesIntegration';
+
+/**
+ * A SvelteKit handle function that wraps the request for Sentry error and
+ * performance monitoring.
+ *
+ * This doesn't currently use OTEL, as it isn't available outside of Node
+ *
+ * Usage:
+ * ```
+ * // src/hooks.server.ts
+ * import { sentryHandle } from '@sentry/sveltekit';
+ *
+ * export const handle = sentryHandle();
+ *
+ * // Optionally use the `sequence` function to add additional handlers.
+ * // export const handle = sequence(sentryHandle(), yourCustomHandler);
+ * ```
+ */
+export function sentryHandle(handlerOptions?: SentryHandleOptions): Handle {
+  const sentryRequestHandler = sentryHandleGeneric(continueTrace, handlerOptions);
+
+  return sentryRequestHandler;
+}
```

Comment:
Why do we need this explicit function instead of re-exporting it like the rest?

Reply:
The Node.js SDK uses `continueTrace` from the Opentelemetry SDK, but that doesn't work in Cloudflare Workers (needs some unsupported Node APIs).

So I moved all the generic code out to the server-common folder, and just pass in the correct `continueTrace` function here.

This also avoids any breaking changes, since the public API doesn't need to change, and the user doesn't need to know anything about `continueTrace`.

Reply:
Ah thanks that makes a lot of sense (also, should have read the JSDoc) 👍